apps/testnorge-arena/arena-core/src/test/java/no/nav/registre/arena/core/provider/rs/BrukereControllerIntegrationTest.java

```diff
@@ -0,0 +1,187 @@
+    private static final String arenaForvalterenUrl="(.*)/arena-forvalteren/api/v1/bruker";
+
+    private SyntetiserArenaRequest syntetiserArenaRequest;
+    private NyeBrukereResponse hentBrukereResponse;
+    private NyeBrukereResponse sendBrukerResponse;
+
+    @Before
+    public void setUp(){
+        syntetiserArenaRequest = new SyntetiserArenaRequest(avspillergruppeId, miljoe, 1);
+
+        hentBrukereResponse = new NyeBrukereResponse();
+        hentBrukereResponse.setAntallSider(0);
+        hentBrukereResponse.setArbeidsoekerList(Collections.emptyList());
+
+        sendBrukerResponse = new NyeBrukereResponse();
+        sendBrukerResponse.setAntallSider(0);
+        sendBrukerResponse.setArbeidsoekerList(Collections.singletonList(Arbeidsoeker.builder()
+                .personident(ident)
+                .miljoe(miljoe)
+                .build()));
+    }
+
+    @Test
+    public void shouldRegistrerBrukerIArenaForvalter() throws Exception {
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(hentLevendeIdenterIAldersgruppeUrl)
+                .withResponseBody(Collections.singletonList(ident))
+                .stubGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(arenaForvalterenUrl)
+                .withResponseBody(hentBrukereResponse)
+                .stubGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(arenaForvalterenUrl)
+                .withResponseBody(sendBrukerResponse)
+                .stubPost();
+
+        var mvcResultat = mvc.perform(post("/api/v1/syntetisering/generer")
+                .queryParam("personident", ident)
+                .content(objectMapper.writeValueAsString(syntetiserArenaRequest))
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(hentLevendeIdenterIAldersgruppeUrl)
+                .withResponseBody(Collections.singletonList(ident))
+                .verifyGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(arenaForvalterenUrl)
+                .withResponseBody(hentBrukereResponse)
+                .verifyGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(arenaForvalterenUrl)
+                .withResponseBody(sendBrukerResponse)
+                .verifyPost();
+
+        NyeBrukereResponse resultat = objectMapper.readValue(mvcResultat,
+                new TypeReference<>() {});
+
+        assertEquals(1, resultat.getArbeidsoekerList().size());
```

Comment:
Jeg anbefaler å teste ut [AssertJ](https://joel-costigliola.github.io/assertj/) for testing. 

Da kan du bruke asserts som assertThat(resultat.getArbeidsoekerList()).hasSize(1).contains(...)

Reply:
God ide, jeg skal bytte til AssertJ 👍